src/components/common/ViewportAwareButton.tsx

```diff
@@ -9,7 +9,15 @@ import { Button } from 'reactstrap';
 /** --------Optional props--------
  * any props you would apply to a reactstrap Button.
  */
-const ViewportAwareButton = function (props) {
+
+interface ViewportAwareButtonProps{
+  icon : React.JSX.Element,
+  text : string,
+  isSmall : boolean,
+  remainder : any,
```

Comment:
Could you remove the space before the colon on each of these 4 lines please? LGTM with that change.

Reply:
Yup!